spec/models/project_step_spec.rb

```diff
@@ -36,4 +36,20 @@
     expect(step[:original_date]).not_to be_nil
   end
 
+  context 'groups' do
```

Comment:
Interesting

On Wed, Sep 14, 2016 at 10:51 AM, Nathan Lee <notifications@github.com>
wrote:

> In spec/models/project_step_spec.rb
> <https://github.com/sassafrastech/madeline_system/pull/136#discussion_r78764467>
> :
>
> > @@ -36,4 +36,20 @@
> >      expect(step[:original_date]).not_to be_nil
> >    end
> >
> > +  context 'groups' do
>
> I have had success using this gem to avoid those extra db hits.
> https://github.com/pcreux/rspec-set
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/136/files/da60517fc880a4b9d0bb4d12052f2d62c0eccedf#r78764467>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cHYnzbHtZh401hJFIrldQeNwKaxhks5qqAnogaJpZM4J6NOi>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech